test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_ontimeinterval.test

```diff
@@ -30,7 +30,22 @@ accumulatorStart
 
 --SHELL--
```

Comment:
Is there any real change in this .test or just cleaning? (In theory, the cache funcationality shouldn't affect to existing functional tests).